opm/simulators/flow/GenericOutputBlackoilModule.cpp

```diff
@@ -493,8 +493,10 @@ assignToSolution(data::Solution& sol)
         DataEntry{"RPORV",    UnitSystem::measure::volume,                                rPorV_},
         DataEntry{"RS",       UnitSystem::measure::gas_oil_ratio,                         rs_},
         DataEntry{"RSSAT",    UnitSystem::measure::gas_oil_ratio,                         gasDissolutionFactor_},
+        DataEntry{"RSW",      UnitSystem::measure::gas_oil_ratio,                         rsw_},
         DataEntry{"RV",       UnitSystem::measure::oil_gas_ratio,                         rv_},
         DataEntry{"RVSAT",    UnitSystem::measure::oil_gas_ratio,                         oilVaporizationFactor_},
+        DataEntry{"RVW",      UnitSystem::measure::oil_gas_ratio,                         rvw_},
```

Comment:
The `RVW` array is an OPM extensions and must never be listed among the "base" solution arrays.  You must find another way to do this.

Reply:
I must admit I don't fully understand why we can not add these arrays to the "base" solution since they are only conditionally added if OPM specific keywords are used. Do you have a suggestion on how this can be done? I really think we should have a solution that makes it possible to restart CO2 runs. For now I have removed the last commit that adds the RSW/RVW arrays. For cases without VAPWAT and DISGASW this is sufficient for restart.

Reply:
> I must admit I don't fully understand why we can not add these arrays to the "base" solution since they are only conditionally added if OPM specific keywords are used.

We've already had far too much trouble with arrays being added to the solution set and breaking compatibility, notably the temperature array, that I don't want to tempt fate. We need to be really careful about these lest we break some production workflows.

> Do you have a suggestion on how this can be done?

Like I said, at minimum we need to remove the `OPM_?WEL` arrays.  Then, and only then, can we begin to explore more specific conditions for these arrays.

> I really think we should have a solution that makes it possible to restart CO2 runs.

Sure, but that has to be a lower priority than keeping existing workflows running.  If you want to do a quick local hack on your machine you can call `restore_wells_ecl()` unconditionally to form the well solution object `xw` in [`RestartIO::load()`](https://github.com/OPM/opm-common/blob/3a107a565f51d5ffd5cd0425ee4749b455bde9fa/opm/output/eclipse/LoadRestart.cpp#L1607-L1609).  If that's enough to enable CO2 simulation restart with `--enable-opm-rst-file=true`, then it should be fairly quick.  If not, then we have more work to do.

Reply:
the hack (https://github.com/OPM/opm-common/pull/4261)  + the changes in the second commit in this PR is sufficient for restart.